src/Autosuggest.js

```diff
@@ -133,7 +133,7 @@ export default class Autosuggest extends Component {
         this.highlightFirstSuggestion();
       }
     } else {
-      if (this.willRenderSuggestions(nextProps)) {
+      if (this.willRenderSuggestions(nextProps, 'suggestions-updated')) {
```

Comment:
Or rather `props-updated`? Or when is `UNSAFE_componentWillReceiveProps` otherwise called? When suggestions update...?

Reply:
Right, suggestions come as a prop... But yeah, this will also be called when other props are changed...

Reply:
The other part of the outer if/else here checks if the suggestions are the same, so that this part of the code actually only runs when the suggestions specifically are updated. It is possible that other props were updated as well, but the suggestions were definitely updated either way, and that's presumably the primary event in this update.